ee/clickhouse/models/team.py

```diff
@@ -0,0 +1,31 @@
+import logging
+from typing import List
+
+from ee.clickhouse.client import sync_execute
+from posthog.settings import CLICKHOUSE_CLUSTER
+
+logger = logging.getLogger(__name__)
+
+# Note: Session recording, dead letter queue, logs deletion will be handled by TTL
+TABLES_TO_DELETE_FROM = [
+    "events",
+    "person",
+    "person_distinct_id",
+    "person_distinct_id2",
+    "groups",
+    "cohortpeople",
+    "person_static_cohort",
+]
+
+
+def delete_teams_data(team_ids: List[int]):
+    logger.info(
+        f"Deleting teams data from clickhouse using background mutations.",
+        team_ids=team_ids,
+        tables=TABLES_TO_DELETE_FROM,
+    )
+    for table in TABLES_TO_DELETE_FROM:
+        sync_execute(
```

Comment:
non-blocking

Is there any chance that there are events etc in kafka that will be added post this run? Do we need to handle this case.

Reply:
There is - we do turn off the api keys, but everything that's not yet ingested would be. 

The answer is likely to delete clickhouse data async in batches on a weekend - something I didn't want to go into this PR.

Reply:
For sure, keep this pr focused on not crashing rather than new features!